src/utils/index.js

```diff
@@ -125,7 +125,7 @@ module.exports.isMobile = function () {
 };
 
 var isIOS = module.exports.isIOS = function () {
-  return /iPad|iPhone|iPod/.test(navigator.platform);
+  return /i(pad|phone|pod)/i.test(navigator.platform);
```

Comment:
I still think that the explicit list of words is more clear than a fancy regular expressions. Each time I have to do any regular expression I have to go and read a tutorial.